content/en/docs/imported/release/notes.md

```diff
@@ -2042,5 +1943,7 @@ Release notes of older releases can be found in:
 - [CHANGELOG-1.7.md](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.7.md)
 - [CHANGELOG-1.8.md](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.8.md)
 - [CHANGELOG-1.9.md](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.9.md)
+- [CHANGELOG-1.10.md](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.10.md)
```

Comment:
Please remove this and the following line. 1.10 should be added after 1.11 ships, and 1.11 shouldn't be in the list at all until 1.12 ships.

Reply:
ok

Reply:
I have remove this, could you review again? thx!